lib/types.ts

```diff
@@ -32,29 +32,84 @@ export interface PluginMetadata {
   imageLayers: string[];
 }
 
-export interface Package {
+export interface PluginResponseStatic extends PluginResponse {
+  hashes: string[];
+}
+
+// NEW STUFF
+
+export interface PluginResponse {
+  plugin: PluginMetadata;
+  scannedProjects: ScannedProjectCustom[];
+}
+
+export interface ScannedProjectCustom {
+  packageManager: string; // actually SupportedPackageManagers; in the CLI
+  depTree: DepTree;
+  targetFile?: string;
+  meta?: any;
+}
+
+export interface DepTreeDep {
+  name: string;
+  version: string;
   dependencies: {
-    [key: string]: any; // TODO
+    [depName: string]: DepTreeDep;
   };
-  docker: {
-    [key: string]: any; // TODO
+  labels?: {
+    [key: string]: string;
   };
-  name: string;
+}
+
+export interface DepTree extends DepTreeDep {
+  type?: string;
   packageFormatVersion: string;
   targetOS: {
     name: string;
     prettyName: string;
     version: string;
   };
-  version: string;
-}
 
-export interface PluginResponse {
-  plugin: PluginMetadata;
-  package: Package;
-  manifestFiles: ManifestFile[];
+  targetFile?: string;
+  policy?: string;
+  docker: {
+    [key: string]: any; // TODO
+  };
+  files?: any;
 }
 
-export interface PluginResponseStatic extends PluginResponse {
-  hashes: string[];
-}
+// export type SupportedPackageManagers =
```

Comment:
Useful as a reference but not sure it should be part of the code! Maybe as part of an internal doc?

Reply:
it's only somewhat useful if it's in this specific place for the next few weeks. afterwards I definitely want it cleaned.